docs/index.md

```diff
@@ -212,42 +212,41 @@ set -gx PATH $PATH ~/.local/bin
 :::
 :::::
 
-
-
 :::::{dropdown} {fa}`info-circle` Why use `pyflyte run` rather than `python example.py`?
 :title: text-muted
 :animate: fade-in-slide-down
 
 `pyflyte run` enables you to execute a specific workflow using the syntax
 `pyflyte run <path/to/script.py> <workflow_or_task_function_name>`.
 
-Keyword arguments can be supplied to ``pyflyte run`` by passing in options in
-the format ``--kwarg value``, and in the case of ``snake_case_arg`` argument
-names, you can pass in options in the form of ``--snake-case-arg value``.
+Keyword arguments can be supplied to `pyflyte run` by passing in options in
+the format `--kwarg value`, and in the case of `snake_case_arg` argument
+names, you can pass in options in the form of `--snake-case-arg value`.
 
 ::::{note}
 If you want to run a workflow with `python example.py`, you would have to write
 a `main` module conditional at the end of the script to actually run the
 workflow:
 
 :::{code-block} python
-if __name__ == "__main__":
-    training_workflow(hyperparameters={"C": 0.1})
+if **name** == "**main**":
```

Comment:
this is incorrect code... was this change auto-generated?

Reply:
I think it is. Let me fix it.